build-infrastructure/codebuild-devbuild-stack.yml

```diff
@@ -339,4 +393,57 @@ Resources:
                   - 's3:GetBucketAcl'
                   - 's3:GetBucketLocation'
           PolicyName: !Sub '${AWS::StackName}-ServicePolicyArm'
-      RoleName: !Sub '${AWS::StackName}-ServiceRoleArm'
\ No newline at end of file
+      RoleName: !Sub '${AWS::StackName}-ServiceRoleArm'
+  ServiceRoleUbuntuArm:
+    Type: 'AWS::IAM::Role'
+    Properties:
+      AssumeRolePolicyDocument:
+        Version: 2012-10-17
+        Statement:
+          - Effect: Allow
+            Principal:
+              Service: codebuild.amazonaws.com
+            Action: 'sts:AssumeRole'
+      Description: Service role, allow access to CW and S3
+      Path: /
+      Policies:
+        - PolicyDocument:
+            Version: 2012-10-17
+            Statement:
+              - Effect: Allow
+                Resource:
+                  - !Join
+                    - ':'
+                    - - 'arn:aws:logs'
+                      - !Ref 'AWS::Region'
+                      - !Ref 'AWS::AccountId'
+                      - !Sub 'log-group:/aws/codebuild/${BuildProjectName}-ubuntu-arm'
```

Comment:
Non-blocking; You can do this all in one `!Sub` command

```
!Sub "arn:aws:logs:${AWS::Region}:${AWS::AccountId}:log-group:/aws/codebuild/${BuildProjectName}-ubuntu-arm"
```

It also makes it a little bit easier to read, I think. There are a couple of other places where this applies as well.